Actions/PullRequestStatusCheck/PullRequestStatusCheck.ps1

```diff
@@ -0,0 +1,17 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $Repository,
+    [Parameter(Mandatory = $true)]
+    [string] $RunId
+)
+
+Write-Host "Checking workflow status for run $RunId in repository $Repository"
+
+$workflowJobs = gh api /repos/$Repository/actions/runs/$RunId/jobs -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" | ConvertFrom-Json
+$failedJobs = $workflowJobs.jobs | Where-Object { $_.conclusion -eq "failure" }
+
+if ($failedJobs) {
+    throw "Workflow failed with the following jobs: $($failedJobs.name -join ', ')"
```

Comment:
Any AL-Go telemetry around here?

Reply:
Ahh, yes, true.

Never mind, then. I thought it'll be nice to track PRs Builds success rate.